Cargo.lock

```diff
@@ -729,6 +729,23 @@ dependencies = [
  "vmm-sys-util 0.4.0 (registry+https://github.com/rust-lang/crates.io-index)",
 ]
 
+[[package]]
+name = "versionize"
+version = "0.1.1"
```

Comment:
I don't get it.. have the other packages also been upgraded? How is this working now?

Reply:
I think the `Cargo.lock` in this PR is stale. Can you double-check with a `cargo build` + `git status`?

Reply:
The other packages are still referencing v0.1.0. The issue is related to only kvm-bindings versioning.

Reply:
Already did that, the cargo.lock file is not stale.